editor/editor_plugin.cpp

```diff
@@ -450,7 +450,7 @@ void EditorPlugin::_bind_methods() {
 	gizmo.return_val.hint = PROPERTY_HINT_RESOURCE_TYPE;
 	gizmo.return_val.hint_string = "EditorSpatialGizmo";
 	ClassDB::add_virtual_method(get_class_static(), gizmo);
-	ClassDB::add_virtual_method(get_class_static(), MethodInfo(Variant::STRING, "get_name"));
+	ClassDB::add_virtual_method(get_class_static(), MethodInfo(Variant::STRING, "_get_name"));
```

Comment:
Is it only used as the main screen plugin label? Maybe it can be simply `get_plugin_name()`.

I don't think `_get_name()` is that bad, considering we have plenty of virtual functions starting with underscore (`_ready`, `_input`, `_process`, etc.).